packages/api/src/routers/projects.ts

```
@@ -55,6 +55,197 @@ export interface DebugPermissionsResult {
+          break;
+        case 'stars':
+          // TODO: Add stars count to project data
+          // For now, fallback to recent
+          orderByClause.push(desc(project.createdAt));
+          break;
+        case 'forks':
+          // TODO: Add forks count to project data
+          // For now, fallback to recent
+          orderByClause.push(desc(project.createdAt));
+          break;
+        case 'recent':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)